src/components/ActionButtons.jsx

```diff
@@ -0,0 +1,57 @@
+/* global _ */
+import React from 'react';
+import classnames from 'classnames';
+
+export default class ActionButtons extends React.Component {
+  constructor(props) {
+    super(props);
+    this.elem = props.elem;
+    this.poi = props.poi;
+    this.favoriteClick = this.favoriteClick.bind(this);
+  }
+
+  favoriteClick() {
+    console.log('hello!');
+    click(this.elem.toggleStorePoi, this.elem);
+  }
+
+  render() {
+    const elem = this.elem;
```

Comment:
According to the previous comment, you should do `const { elem, poi } = this.props;` instead.